--- adie/TextWindow.cpp.str	2009-08-22 04:00:00.000000000 -0400
+++ adie/TextWindow.cpp	2009-08-22 04:00:20.000000000 -0400
@@ -1421,7 +1421,7 @@
 // Reopen file
 long TextWindow::onCmdReopen(FXObject*,FXSelector,void*){
   if(isModified()){
-    if(FXMessageBox::question(this,MBOX_YES_NO,tr("Document was changed"),tr("Discard changes to this document?"))==MBOX_CLICKED_NO) return 1;
+    if(FXMessageBox::question(this,MBOX_YES_NO,tr("Document was changed"),"%s",tr("Discard changes to this document?"))==MBOX_CLICKED_NO) return 1;
     }
   loadFile(filename);
   return 1;
--- src/FXGLViewer.cpp.str	2009-08-22 03:55:01.000000000 -0400
+++ src/FXGLViewer.cpp	2009-08-22 03:56:01.000000000 -0400
@@ -2420,7 +2420,7 @@
 
       // Try open printer
       if(!pdc.beginPrint(printer)){
-        FXMessageBox::error(this,MBOX_OK,tr("Printer Error"),tr("Unable to print."));
+        FXMessageBox::error(this,MBOX_OK,tr("Printer Error"),"%s",tr("Unable to print."));
         return 1;
         }
 
@@ -2595,7 +2595,7 @@
     dlg.getPrinter(printer);
     FXDCPrint pdc(getApp());
     if(!pdc.beginPrint(printer)){
-      FXMessageBox::error(this,MBOX_OK,tr("Printer Error"),tr("Unable to print."));
+      FXMessageBox::error(this,MBOX_OK,tr("Printer Error"),"%s",tr("Unable to print."));
       return 1;
       }
 
